recipe/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "python-ffmpeg" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4d5275a58da2a82fb550acbf496ff86e76d677e06b781baeefbe05a4cd47ab50
+
+build:
+  # ffmpeg is not available for s390x
+  skip: true  # [s390x]
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - python
+    - pyee
+    - typing_extensions
+    - ffmpeg
+
+test:
+  requires:
+    - pip
+  imports:
+    - ffmpeg
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/jonghwanhyeon/python-ffmpeg
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: |
+    A python binding for FFmpeg which provides sync and async APIs
+  description: |
```

Comment:
I usually try to make the description a little more verbose than the summary

Reply:
I agree. Unfortunately, _"A python binding for FFmpeg which provides sync and async APIs"_ is the longest sentence in the whole repo and docs 🌲